vnext/ReactUWP/Views/DynamicAutomationPeer.cpp

```diff
@@ -248,8 +269,48 @@ void DynamicAutomationPeer::Collapse() const {
   DynamicAutomationProperties::DispatchAccessibilityAction(Owner(), L"collapse");
 }
 
-// Private Methods
+// IRangeValueProvider
+double DynamicAutomationPeer::Minimum() {
+  return GetAccessibilityValueRange(winrt::PROJECT_ROOT_NAMESPACE::AccessibilityValue::Min);
+}
+
+double DynamicAutomationPeer::Maximum() {
+  return GetAccessibilityValueRange(winrt::PROJECT_ROOT_NAMESPACE::AccessibilityValue::Max);
+}
+
+double DynamicAutomationPeer::Value() {
+  return GetAccessibilityValueRange(winrt::PROJECT_ROOT_NAMESPACE::AccessibilityValue::Now);
+}
+
+// Controls such as Slider, ProgressBar, ScrollBar are by definition interactive.
+bool DynamicAutomationPeer::IsReadOnly() {
+  return false;
+}
 
+void DynamicAutomationPeer::SetValue(double value) {
+  try {
+    if (auto const &owner = Owner()) {
+      DynamicAutomationProperties::DispatchAccessibilityAction(owner, L"setValue");
+    }
+  } catch (...) {
+  }
+}
+
+// Doesn't have a React Native analog.
+// Return default value for RangeBase-derived types, such as Slider and ProgressBar.
+// https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.primitives.rangebase.smallchange?view=winrt-19041
+double DynamicAutomationPeer::SmallChange() {
+  return 0.1;
```

Comment:
from the doc:
`Slider overrides the metadata of this property and sets its default to 1`

I think you want to get the value from the xaml elemental type?

Reply:
Per what Jon has told me: this code doesn't get invoked for controls such as the RN <Slider> - that one has its own automation peer if it is implemented using the XAML Slider. The DynamicAutomationPeer handles custom controls people write in RN using the <View> tag as their basis, and mark their accessibilityRole as XYZ, meaning that custom control is supposed to **fulfill the role of and act like XYZ** but it doesn't actually use any similar-looking XAML controls. 

This means, I cannot cast the Owner() to RangeBase, because the RN element is a ViewControl/ViewPanel and not a RangeBase underneath.

Reply:
What I could do, though, is to check the accessibilityRole of the owner, and if it's an adjustable (aka slider) I could return 1, instead of 0.1.

Reply:
I don't know why the page didn't show your comment while I was typing mine..
@jonthysell Can you say whether my latest update to Small/LargeChange() does what you intended? You said "shouldn't depend on the kind of Slider" - I added a special case for the Slider, while leaving the previous default for the rest. Is that ok?

Reply:
No this shouldn't rely on the type Slider or any particular type, it should ask the runtime type for the default value for the property

Reply:
But the runtime type doesn't have this property, since it's just a panel, right?